app/_data/tables/support/gateway/packages.yml

```diff
@@ -5,105 +5,123 @@ _packages:
     package: true
     arm: false
     fips: false
+    graviton: false
   amazonlinux1: &amazonlinux1
     os: Amazon Linux
     version: 1
     package: true
     arm: false
     fips: false
+    graviton: false
   amazonlinux2: &amazonlinux2
     os: Amazon Linux
     version: 2
     package: true
     arm: false
     fips: false
+    graviton: true
   amazonlinux2022: &amazonlinux2022
     os: Amazon Linux
     version: 2022
     package: true
     arm: false
     fips: false
+    graviton: false
   centos7: &centos7
     os: CentOS
     version: 7
     package: true
     arm: false
     fips: false
+    graviton: false
   centos8: &centos8
     os: CentOS
     version: 8
     package: true
     arm: false
     fips: false
+    graviton: false
   debian8: &debian8
     os: Debian
     version: 8
     package: true
     arm: false
     fips: false
+    graviton: false
   debian9: &debian9
     os: Debian
     version: 9
     package: true
     arm: false
     fips: false
+    graviton: false
   debian10: &debian10
     os: Debian
     version: 10
     package: true
     arm: false
     fips: false
+    graviton: false
   debian11: &debian11
     os: Debian
     version: 11
     package: true
     arm: false
     fips: false
+    graviton: false
   debian-stable: &debian-stable
     os: Debian
     version: stable
     package: true
     arm: false
     fips: false
+    graviton: false
   rhel7: &rhel7
     os: RHEL
     version: 7.x
     package: true
     arm: false
     fips: false
+    graviton: false
   rhel8: &rhel8
     os: RHEL
     version: 8.x
     package: true
     arm: false
     fips: false
+    graviton: false
   rhel9: &rhel9
     os: RHEL
     version: 9.x
     package: true
     arm: false
     fips: false
+    graviton: false
   ubuntu1604: &ubuntu1604
     os: Ubuntu
     version: 16.04
     package: true
     arm: false
     fips: false
+    graviton: false
   ubuntu1804: &ubuntu1804
     os: Ubuntu
     version: 18.04
     package: true
     arm: false
     fips: false
+    graviton: true
```

Comment:
this is not correct - for Kong to support graviton for a particular linux flavor - Kong has to have ARM support for that flavor.

said another way - if Kong supports ARM for a flavor, it supports Graviton.